Assignment2/juanbm99-fromRDFtoJSON-LD.jsonld

```diff
@@ -0,0 +1,59 @@
+{
+	"@context": {
+			"vocab": "http://dispositivos.com#",
+			"xsd": "http://www.w3.org/2001/XMLSchema#",
+			"Class01": "vocab:Class01",
+			"Sensor029": "Class01:Sensor029",
+			"Computer101": "Class01:Computer101",
+			"Measurement8401": "Sensor029:Measurement8401",
+			"User10A": "Computer101:User10A",
+			"includes": "vocab:includes",
+			"hasMeasurement": "vocab:hasMeasurement",
+			"hasTemperature": "vocab:hasTemperature",
+			"atTime":"vocab:atTime",
+			"hasOwner": "vocab:hasOwner",
+			"hasName": "vocab:hasName"
+		},
+	"Class01": {
+		"@id": "http://dispositivos.com/Class01",
+		"@type": "vocab:Class01",
+		"includes": [
+			{
+				"Sensor029":{
+					"@id": "http://dispositivos.com/Class01/Sensor029",
+					"@type": "Class01:Sensor029",
```

Comment:
This type definition works, but semantically isn't ok. The best way to define it might be defining using your prefix re and the concept name, e.g. "@type": "re:Sensor".
This comment also applies for the other type definitions. You don't need to change it, but remember to provide good  type definitions for the future.